src/main/webapp/modal-overlay.css

```diff
@@ -36,4 +36,14 @@
 
 .modal-body {
   padding: 10px 15px;
+}
+
+#overlay {
+  position: fixed;
+  /* Unsure how to shorthand these */
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
```

Comment:
is zero not the default value for these? otherwise it's fine, this works

Reply:
I believe so. I was leaving it verbose to be more thorough.

Reply:
Nevermind. It's important to have those there.
Reinstated 82e8aef